src/lib/coda_networking/coda_networking.ml

```diff
@@ -337,7 +337,9 @@ module Rpcs = struct
       let name = "get_ancestry"
 
       module T = struct
-        type query = Consensus.Data.Consensus_state.Value.t
+        (** NB: The state hash sent in this query should not be trusted, as it can be forged. This is ok for how this RPC is implented, as we only use the state hash for tie breaking when checking whether or not the proof is worth serving. *)
```

Comment:
Why do we send the state hash? Shouldn't we just compute it once when we receive the block?

Reply:
We don't send the entire block, and the state hash isn't actually important outside of a comparison to see if the query is worth responding to. I originally wrote this to send a full block instead of a consensus state, but I changed it back to a consensus state and just a hash since a block can actually be pretty large (and we don't inspect any contents of it beyond the consensus state, and now, the hash).